packages/ui/src/components/wui-shimmer/index.ts

```diff
@@ -0,0 +1,31 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import styles from './styles'
+import type { BorderRadiusType } from '../../utils/TypesUtil'
+
+@customElement('wui-shimmer')
+export class WuiShimmer extends LitElement {
+  public static styles = [styles]
+
+  @property() public width = ''
+
+  @property() public height = ''
+
+  @property() public borderRadius: BorderRadiusType = 'm'
+
+  public render() {
+    this.style.cssText = `
+      width: ${this.width};
+      height: ${this.height};
+      border-radius: ${`var(--wui-border-radius-${this.borderRadius})`};
+    `
+
+    return html``
```

Comment:
I guess there is no way to get around this in shimmers scenario? Like, can it return null or something?

Reply:
`return null` works, so changed it to that